### PR TITLE
VBM-80: Fix logo sign-in sign-up pages 

### PR DESCRIPTION
**Pull request by Val-M9**
added whiteLogo component for header since it always white on flag background. Change collorPallette's yellow/blue colors

### CODE REVIEW

File: packages/mobile/src/components/logo/logo-white.tsx [hunk 1]
```diff
@@ -0,0 +1,58 @@
+import React from 'react';
+import Svg, { SvgProps, Path } from 'react-native-svg';
+import { useCustomTheme } from '~/hooks/hooks';
+
+const LogoWhite = (props: SvgProps) => {
+  const { colors } = useCustomTheme();
```
Comment (AleksandrTermenzhy) [inline packages/mobile/src/components/logo/logo-white.tsx:6]: White is just... white)) We don't need different colors for it

File: packages/mobile/src/config/navigation-theme.config.ts [hunk 2]
```diff
@@ -48,9 +49,10 @@ const NavigationDarkTheme = {
     icon: ColorPalette.WHITE_100,
     titlePrimary: ColorPalette.WHITE_100,
     titleSecondary: ColorPalette.WHITE_100,
-    yellow: ColorPalette.WHITE_100,
-    blue: ColorPalette.WHITE_100,
+    yellow: ColorPalette.YELLOW_200,
+    blue: ColorPalette.BLUE_100,
     subtitle: ColorPalette.YELLOW_100,
+    logoWhite: ColorPalette.WHITE_100,
```
Comment (AleksandrTermenzhy) [inline packages/mobile/src/config/navigation-theme.config.ts:55]: Remove all of these

**Review by AleksandrTermenzhy — CHANGES_REQUESTED** [review]

File: packages/mobile/src/components/logo/logo-white.tsx (hunk 1 above)
Comment (Val-M9) [reply packages/mobile/src/components/logo/logo-white.tsx:6]: fixed ))

File: packages/mobile/src/config/navigation-theme.config.ts (hunk 2 above)
Comment (Val-M9) [reply packages/mobile/src/config/navigation-theme.config.ts]: delete logoWhite, but leave blue and yellow, as it maybe in use somewhere

**Comment by Val-M9** [comment]
[welcome-signin-signup - 26.08.2022, 08_46_37.webm](https://user-images.githubusercontent.com/71255723/186831161-02343cd1-679b-4eee-a9fc-d8dca8cc57ae.webm)

File: packages/mobile/src/components/logo/logo-white.tsx [hunk 3]
```diff
@@ -0,0 +1,58 @@
...
+
+  return (
+    <Svg width={180} height={44} viewBox="0 0 1198 298" fill="none" {...props}>
+      <Path
+        d="M411.456 232.629L363.364 92.9743H384.762L418.871 193.06L421.625 218.029H425.862L432.853 230.936C429.605 231.5 426.074 231.923 422.26 232.206C418.588 232.629 414.987 232.77 411.456 232.629ZM379.889 106.305L350.652 102.919L350.864 92.9743H380.524L379.889 106.305ZM432.853 230.936L420.777 223.53L466.115 92.9743H482.852L432.853 230.936ZM387.728 107.151L383.279 92.9743H407.43L408.702 99.7454C405.594 101.156 402.205 102.496 398.532 103.766C394.86 104.894 391.259 106.023 387.728 107.151ZM471.835 106.305L442.599 102.919L442.811 92.9743H472.471L471.835 106.305ZM473.742 107.151L469.293 92.9743H493.445L494.716 99.7454C491.609 101.156 488.219 102.496 484.547 103.766C480.875 104.894 477.273 106.023 473.742 107.151Z"
+        fill={colors.whiteColor}
```
Comment (AleksandrTermenzhy) [inline packages/mobile/src/components/logo/logo-white.tsx:12]:
```suggestion
        fill={colors.ColorPalette.WHITE_100}
```

**Review by AleksandrTermenzhy — APPROVED** [review]

File: packages/mobile/src/components/logo/logo-white.tsx (hunk 3 above)
Comment (Val-M9) [reply packages/mobile/src/components/logo/logo-white.tsx]: fixed

**Pull request merged**